main.c

```diff
@@ -68,11 +25,19 @@ void testPairToNumber(
 }
 
 int main() {
+
+    ColorPair colorPair;
+    printf("PRINTING MANUAL\n");
```

Comment:
Color code manual printing is a functionality of the software and hence can be moved to a more related file as compared to main.